src/sagas/login.js

```diff
@@ -175,11 +179,15 @@ function* checkUserFilterPermissions () {
 }
 
 function* initialLoad () {
+  const userId = yield select(state => state.config.getIn(['user', 'id']))
+  const filters = JSON.parse(loadFromLocalStorage(`vmFilters-${userId}`)) || {}
+
   // no data prerequisites
   yield all([
     call(fetchUserGroups, getUserGroups()),
     call(fetchAllOS, getAllOperatingSystems()),
     call(fetchAllHosts, getAllHosts()),
+    call(saveFilters, saveVmsFilters({ filters })),
```

Comment:
Thanks for moving it over here.  Would be nice to have a `loadFiltersFromLocalStorage` or similar saga function to just to keep it all in one place.